ui_testing/pages/order_page.py

```diff
@@ -125,13 +125,7 @@ def create_existing_order(self, no='', material_type='', article='', contact='',
         
         for test_unit in test_units:
             self.set_test_unit(test_unit)    
-        if multiple_suborders > 0:
```

Comment:
i changed it so i can create any amount of orders at the same time, because all my test cases where on the 2nd or 3rd suborder, so it was much easier to create multiple suborders and keep track of them using this method